<html>
<head>
<title>TestNG:  CSVloginIOS</title>
<link href="../testng.css" rel="stylesheet" type="text/css" />
<link href="../my-testng.css" rel="stylesheet" type="text/css" />

<style type="text/css">
.log { display: none;} 
.stack-trace { display: none;} 
</style>
<script type="text/javascript">
<!--
function flip(e) {
  current = e.style.display;
  if (current == 'block') {
    e.style.display = 'none';
    return 0;
  }
  else {
    e.style.display = 'block';
    return 1;
  }
}

function toggleBox(szDivId, elem, msg1, msg2)
{
  var res = -1;  if (document.getElementById) {
    res = flip(document.getElementById(szDivId));
  }
  else if (document.all) {
    // this is the way old msie versions work
    res = flip(document.all[szDivId]);
  }
  if(elem) {
    if(res == 0) elem.innerHTML = msg1; else elem.innerHTML = msg2;
  }

}

function toggleAllBoxes() {
  if (document.getElementsByTagName) {
    d = document.getElementsByTagName('div');
    for (i = 0; i < d.length; i++) {
      if (d[i].className == 'log') {
        flip(d[i]);
      }
    }
  }
}

// -->
</script>

</head>
<body>
<h2 align='center'>CSVloginIOS</h2><table border='1' align="center">
<tr>
<td>Tests passed/Failed/Skipped:</td><td>0/1/0</td>
</tr><tr>
<td>Started on:</td><td>Mon Dec 18 21:45:41 IST 2017</td>
</tr>
<tr><td>Total time:</td><td>15 seconds (15061 ms)</td>
</tr><tr>
<td>Included groups:</td><td></td>
</tr><tr>
<td>Excluded groups:</td><td></td>
</tr>
</table><p/>
<small><i>(Hover the method name to see the test class name)</i></small><p/>
<table width='100%' border='1' class='invocation-failed'>
<tr><td colspan='4' align='center'><b>FAILED TESTS</b></td></tr>
<tr><td><b>Test method</b></td>
<td width="30%"><b>Exception</b></td>
<td width="10%"><b>Time (seconds)</b></td>
<td><b>Instance</b></td>
</tr>
<tr>
<td title='automationAssignment.CSVloginIOS.test()'><b>test</b><br>Test class: automationAssignment.CSVloginIOS<br>Parameters: 60000, false, 5</td>
<td><div><pre>com.experitest.client.InternalException: Exception caught while executing startLoggingDevice: Failed to start logging for device: iPhone 4. see SeeTest log for details
	at com.experitest.client.JavaClientInternals.executeInternally(JavaClientInternals.java:249)
	at com.experitest.client.Client.execute(Client.java:247)
	at com.experitest.client.Client.startLoggingDevice(Client.java:2266)
	at automationAssignment.CSVloginIOS.loginTestIOS(CSVloginIOS.java:38)
	at automationAssignment.CSVloginIOS.testAux(CSVloginIOS.java:26)
	at automationAssignment.BaseTest.test(BaseTest.java:120)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 16 stack frames</pre></div><a href='#' onClick='toggleBox("stack-trace902702466", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace902702466'><pre>com.experitest.client.InternalException: Exception caught while executing startLoggingDevice: Failed to start logging for device: iPhone 4. see SeeTest log for details
	at com.experitest.client.JavaClientInternals.executeInternally(JavaClientInternals.java:249)
	at com.experitest.client.Client.execute(Client.java:247)
	at com.experitest.client.Client.startLoggingDevice(Client.java:2266)
	at automationAssignment.CSVloginIOS.loginTestIOS(CSVloginIOS.java:38)
	at automationAssignment.CSVloginIOS.testAux(CSVloginIOS.java:26)
	at automationAssignment.BaseTest.test(BaseTest.java:120)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:100)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:646)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:811)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1129)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:746)
	at org.testng.TestRunner.run(TestRunner.java:600)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.access$000(SuiteRunner.java:39)
	at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:400)
	at org.testng.internal.thread.ThreadUtil$2.call(ThreadUtil.java:64)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
</pre></div></td>
<td>12</td>
<td>automationAssignment.CSVloginIOS@4516af24</td></tr>
</table><p>
</body>
</html>